src/pages/edit_resource/access_and_licence_step/LicenseWizardFields.tsx

```diff
@@ -25,6 +27,8 @@ const modifyAndBuildRadio = 'change-and-build';
 enum DefaultRestricion {
   CC_BY = 'CC BY 4.0',
   yes = 'yes',
+  NTNU = 'ntnu-internt',
```

Comment:
disse går igjen - kanskje vi kunne flyttet konstantene til et sted for felles bruk ? kanskje under license.types.ts ?